assets/js/edit.js

```diff
@@ -285,7 +341,31 @@ function Edit( props ) {
 								'simple-podcasting'
 							)}
 							checked={displayEpisodeType}
-							onChange={() => setAttributes({ displayEpisodeType: !displayEpisodeType})}
+							onChange={() => setAttributes({ displayEpisodeType: !displayEpisodeType })}
+						/>
+					</PanelRow>
+					<PanelRow>
+						<RadioControl
+							label={__('Dock Player', 'simple-podcasting')}
+							help={__('This will only affect the display of the player on the frontend.', 'simple-podcasting')}
```

Comment:
The details are currently replaced with the `More` button.

If I've presumed intent incorrectly, let me know and you can reverse the fix and remove the more button in the editor :)

```suggestion
							help={__('This will only affect the position of the player on the frontend.', 'simple-podcasting')}
```

With top or bottom selected in the editor:
<img width="657" height="211" alt="Screenshot 2025-09-08 at 1 17 40 pm" src="https://github.com/user-attachments/assets/0a45b0b1-fb76-45cc-b651-0a42f1e26262" />

Reply:
@peterwilsoncc Thanks! I have removed related code for the editor :)